frame/vault/src/lib.rs

```diff
@@ -487,6 +490,26 @@ pub mod pallet {
 			Self::deposit_event(Event::VaultStarted { vault });
 			Ok(().into())
 		}
+		#[pallet::weight(10_000)]
+		pub fn liquidate_strategy(
+			origin: OriginFor<T>,
+			vault_idx: VaultIndex,
+			strategy_account: T::AccountId,
+		) -> DispatchResult {
+			let from = ensure_signed(origin)?;
+			ensure!(Self::vault_info(&vault_idx)?.manager == from, Error::<T>::AccountIsNotManager);
```

Comment:
Most parachains use the Collective/Membership pallet to deal with stuff like this

Reply:
like EnsureRootOrHalfCouncil

Reply:
Yes but I am just saying. Still not sure if it is possible to integrate them with the Vault infrastructure